editor/export/project_export.cpp

```diff
@@ -1395,12 +1395,14 @@ ProjectExportDialog::ProjectExportDialog() {
 
 	set_cancel_button_text(TTR("Close"));
 	set_ok_button_text(TTR("Export PCK/ZIP..."));
+	get_ok_button()->set_tooltip_text("Export the project resources as a PCK or ZIP package. This is not a playable build, only the project data without a Godot executable.");
```

Comment:
Forgot this before (same for the other tooltip), this should use `TTR` to be extracted for translation.
```suggestion
	get_ok_button()->set_tooltip_text(TTR("Export the project resources as a PCK or ZIP package. This is not a playable build, only the project data without a Godot executable."));
```

Reply:
Good catch. Sorry for the miss.
On the bright side, you got your co-authorship back in the latest commit ^^
Thx